Source/Core/DolphinQt/GBAWidget.cpp

```diff
@@ -229,6 +229,36 @@ void GBAWidget::DoState(bool export_state)
       false);
 }
 
+void GBAWidget::ImportExportSave(bool export_save)
```

Comment:
I don't personally have a strong opinion about this but I know some people prefer an enumeration over a boolean

Reply:
I just copied the existing `DoState()` more or less, but yeah, probably.